src/test/java/stepDefinition/WishListPageStepDef.java

```diff
@@ -1,29 +1,34 @@
 package stepDefinition;
 
+import static org.hamcrest.CoreMatchers.is;
+
 import cucumber.api.java.en.Then;
-import org.junit.Assert;
 import pages.WishListPage;
 import utils.ScenarioContext;
 
+import static utils.AssertUtils.assertThat;
+import static utils.AssertUtils.assertTrue;
+
 public class WishListPageStepDef extends AbstractStepDef {
-    WishListPage wishListPage = new WishListPage(webDriver);
+
+    private WishListPage wishListPage = new WishListPage(webDriver);
 
     @Then("^product '(.*)' is displayed in the WishList$")
-    public void displayProductWishlist(String expectedItem) {
+    public void displayProductWishList(String expectedItem) {
         waitForPageLoaded();
-        Assert.assertEquals("The product is not found in the Wishlist", expectedItem, wishListPage.getProductName().getText());
+        assertThat("The product is found in the WishList", expectedItem, is(wishListPage.getProductName().getText()));
     }
 
     @Then("^updated product quantity is displayed in the WishList$")
     public void updatedProductQuantityIsDisplayedInTheWishList() {
         waitVisibility(wishListPage.getQuantityField());
         String actual = wishListPage.getQuantityField().getAttribute("value");
-        Assert.assertEquals("Quantity does not match", ScenarioContext.getValue("quantity"), actual);
+        assertThat("Quantity does not match", ScenarioContext.getValue("quantity"), is(actual));
     }
 
     @Then("^product is removed successfully from Wishlist$")
     public void productIsRemovedSuccessfully() {
         waitVisibility(wishListPage.getEmptyWishlistMessage());
-        Assert.assertEquals("The wishlist is empty!", wishListPage.getEmptyWishlistMessage().getText());
+        assertTrue("The wishlist is empty", "The wishlist is empty!".equals(wishListPage.getEmptyWishlistMessage().getText()));
```

Comment:
Change to assert equals please
Or use assertUtils

Reply:
done, thank you!